### PR TITLE
dev-python/neovim-remote:  python3_{9..10} support

### PR DESCRIPTION
**Pull request by dAgbeve**
Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Douglas Agbeve <douglas@agbeve.com>

### CODE REVIEW

File: dev-python/neovim-remote/neovim-remote-2.4.0.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7,8,9} )
 DISTUTILS_USE_SETUPTOOLS=rdepend
```
Comment (a17r) [inline dev-python/neovim-remote/neovim-remote-2.4.0.ebuild:7]: Should have an empty line after `EAPI` instead, but not necessarily before `inherit`.
Tests pass?
What about python3_10?

File: dev-python/neovim-remote/neovim-remote-9999.ebuild [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-PYTHON_COMPAT=( python3_{7,8} )
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8,9} )
```
Comment (a17r) [inline dev-python/neovim-remote/neovim-remote-9999.ebuild:6]: Should have an empty line after `EAPI` instead, but not necessarily before `inherit`.
Tests pass?
What about python3_10?

**Review by a17r — CHANGES_REQUESTED** [review]

File: dev-python/neovim-remote/neovim-remote-9999.ebuild (hunk 2 above)
Comment (dAgbeve) [reply dev-python/neovim-remote/neovim-remote-9999.ebuild:6]: Tests passed, currently working on python3_10.

**Review by a17r — COMMENTED** [review]
`pynvim/pynvim: python3_10 support` - could it be that you are not using `repoman commit` for your commits? ;)

In general it would be nice to always add a little snippet "Tests passed" to git summary when you are bumping python targets on foreign packages.

**Comment by dAgbeve** [comment]
> `pynvim/pynvim: python3_10 support` - could it be that you are not using `repoman commit` for your commits? ;)
> 
> In general it would be nice to always add a little snippet "Tests passed" to git summary when you are bumping python targets on foreign packages.

to meet QA on `dev-python/neovim-remote` i had to bump `pynvim/pynvim` to python3_10 and yea i always use repoman commit.

Good to know that i had to add a snippet of `Test passed` , didnt know and since i had not seen it from other commits.




==============

platform linux -- Python 3.10.0b1, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /usr/bin/python3.10
cachedir: .pytest_cache
rootdir: /var/tmp/portage/dev-python/pynvim-0.4.3/work/pynvim-0.4.3, configfile: setup.cfg, testpaths: test
collecting ... collected 69 items

test/test_buffer.py::test_repr PASSED                                      [  1%]
test/test_buffer.py::test_get_length PASSED                                [  2%]
test/test_buffer.py::test_get_set_del_line PASSED                          [  4%]
test/test_buffer.py::test_get_set_del_slice PASSED                         [  5%]
test/test_buffer.py::test_vars PASSED                                      [  7%]
test/test_buffer.py::test_api PASSED                                       [  8%]
test/test_buffer.py::test_options PASSED                                   [ 10%]
test/test_buffer.py::test_number PASSED                                    [ 11%]
test/test_buffer.py::test_name PASSED                                      [ 13%]
test/test_buffer.py::test_valid PASSED                                     [ 14%]
test/test_buffer.py::test_append PASSED                                    [ 15%]
test/test_buffer.py::test_mark PASSED                                      [ 17%]
test/test_buffer.py::test_invalid_utf8 PASSED                              [ 18%]
test/test_buffer.py::test_get_exceptions PASSED                            [ 20%]
test/test_buffer.py::test_set_items_for_range PASSED                       [ 21%]
test/test_buffer.py::test_update_highlights PASSED                         [ 23%]
test/test_buffer.py::test_buffer_inequality PASSED                         [ 24%]
test/test_client_rpc.py::test_call_and_reply PASSED                        [ 26%]
test/test_client_rpc.py::test_call_api_before_reply PASSED                 [ 27%]
test/test_client_rpc.py::test_async_call PASSED                            [ 28%]
test/test_client_rpc.py::test_recursion PASSED                             [ 30%]
test/test_concurrency.py::test_interrupt_from_another_thread PASSED        [ 31%]
test/test_concurrency.py::test_exception_in_threadsafe_call PASSED         [ 33%]
test/test_decorators.py::test_command_count PASSED                         [ 34%]
test/test_events.py::test_receiving_events PASSED                          [ 36%]
test/test_events.py::test_sending_notify PASSED                            [ 37%]
test/test_events.py::test_async_error PASSED                               [ 39%]
test/test_events.py::test_broadcast PASSED                                 [ 40%]
test/test_host.py::test_host_imports PASSED                                [ 42%]
test/test_host.py::test_host_clientinfo PASSED                             [ 43%]
test/test_host.py::test_host_async_error PASSED                            [ 44%]
test/test_logging.py::test_setup_logging PASSED                            [ 46%]
test/test_tabpage.py::test_windows PASSED                                  [ 47%]
test/test_tabpage.py::test_vars PASSED                                     [ 49%]
test/test_tabpage.py::test_valid PASSED                                    [ 50%]
test/test_tabpage.py::test_number PASSED                                   [ 52%]
test/test_tabpage.py::test_repr PASSED                                     [ 53%]
test/test_vim.py::test_clientinfo PASSED                                   [ 55%]
test/test_vim.py::test_command PASSED                                      [ 56%]
test/test_vim.py::test_command_output PASSED                               [ 57%]
test/test_vim.py::test_command_error PASSED                                [ 59%]
test/test_vim.py::test_eval PASSED                                         [ 60%]
test/test_vim.py::test_call PASSED                                         [ 62%]
test/test_vim.py::test_api PASSED                                          [ 63%]
test/test_vim.py::test_strwidth PASSED                                     [ 65%]
test/test_vim.py::test_chdir PASSED                                        [ 66%]
test/test_vim.py::test_current_line PASSED                                 [ 68%]
test/test_vim.py::test_current_line_delete PASSED                          [ 69%]
test/test_vim.py::test_vars PASSED                                         [ 71%]
test/test_vim.py::test_options PASSED                                      [ 72%]
test/test_vim.py::test_local_options PASSED                                [ 73%]
test/test_vim.py::test_buffers PASSED                                      [ 75%]
test/test_vim.py::test_windows PASSED                                      [ 76%]
test/test_vim.py::test_tabpages PASSED                                     [ 78%]
test/test_vim.py::test_hash PASSED                                         [ 79%]
test/test_vim.py::test_cwd FAILED                                          [ 81%]
test/test_vim.py::test_lua PASSED                                          [ 82%]
test/test_window.py::test_buffer PASSED                                    [ 84%]
test/test_window.py::test_cursor PASSED                                    [ 85%]
test/test_window.py::test_height PASSED                                    [ 86%]
test/test_window.py::test_width PASSED                                     [ 88%]
test/test_window.py::test_vars PASSED                                      [ 89%]
test/test_window.py::test_options PASSED                                   [ 91%]
test/test_window.py::test_position PASSED                                  [ 92%]
test/test_window.py::test_tabpage PASSED                                   [ 94%]
test/test_window.py::test_valid PASSED                                     [ 95%]
test/test_window.py::test_number PASSED                                    [ 97%]
test/test_window.py::test_handle PASSED                                    [ 98%]
test/test_window.py::test_repr PASSED                                      [100%]

========

**Comment by a17r** [comment]
> I had to bump `pynvim/pynvim` to python3_10 and yea i always use repoman commit.

But `pynvim/pynvim` doesn't exist and repoman certainly didn't prepare it that way. ;)
`dev-python/pynvim` on the other hand exists.

> Good to know that i had to add a snippet of `Test passed` , didnt know and since i had not seen it from other commits.

That's just good etiquette so that it is clear from the package maintainer POV.

**Review by a17r — CHANGES_REQUESTED** [review]
Also, your `Signed-off-by` line is broken on all commits, lacks angle brackets.

**Comment by dAgbeve** [comment]
> Also, your `Signed-off-by` line is broken on all commits, lacks angle brackets.

Thank so much for your review and attention to detail. All set now :)

**Pull request closed without merging**